.github/workflows/package.yml

```
@@ -0,0 +1,95 @@
+name: package
+on: [push, pull_request]
+jobs:
```

```suggestion

jobs:
```

(Maybe, to make things more "balanced.")